### ISSUE TITLE
[9.4.1] Server Crashes with LiteDB ENSURE Error

### ISSUE THREAD

**Issue by epeterson12** [StrangeLoopGames/EcoIssues#22684]
From the starting game menu, when I try to host a server, the setup starts normally, then fails with the following message in a popup: `The server crashed with error 'LiteDB ENSURE: page type must be collections page' and you can't play. Check logs in 'Eco_Data\Server\Logs' for more information`. Here is the log file: [log_210920050700.log](https://github.com/StrangeLoopGames/EcoIssues/files/7199581/log_210920050700.log).

I have been using the server (running in Windows) since January 2021, but I started getting the error message 3 days ago.

**Comment by Daniel-Magulado** [comment]
Hi @epeterson12 

Thank you for writing this report. With the latest official game version of 0.9.4.3, are you still encountering this issue?

**Comment by epeterson12** [comment]
Yes, I'm still getting the same error message with version 0.9.4.3.
[log_210926121945.log](https://github.com/StrangeLoopGames/EcoIssues/files/7232487/log_210926121945.log)

**Comment by Daniel-Magulado** [comment]
Hi @epeterson12 

Can you provide us your saved game file? So we can look into it further? Here's the instruction: https://github.com/StrangeLoopGames/EcoIssues/wiki/Uploading-a-world-save

**Comment by epeterson12** [comment]
Here is a link to my saved game file: https://usherbrooke-my.sharepoint.com/:u:/g/personal/pete3002_usherbrooke_ca/EX6p9BV8fllClBTMC3zaiNMBqv1_0t0RmhE4CcTMiTmgLQ?e=RckmJ9
